src/main/kotlin/io/snyk/plugin/SnykPostStartupActivity.kt

```diff
@@ -33,6 +35,14 @@ class SnykPostStartupActivity : StartupActivity.DumbAware {
         if (!ApplicationManager.getApplication().isUnitTestMode) {
             project.service<SnykTaskQueueService>().downloadLatestRelease()
         }
+
+        val twoWeeksInSeconds: Long = 14 * 24 * 60 * 60
+        val feedbackRequestShownMoreThenTwoWeeksAgo =
+            settings.lastTimeFeedbackRequestShown.toInstant().plusSeconds(twoWeeksInSeconds).isBefore(Instant.now())
```

Comment:
How about removing `twoWeeksInSeconds` variable and use [plus](https://docs.oracle.com/javase/8/docs/api/java/time/Instant.html#plus-long-java.time.temporal.TemporalUnit-) method with time unit for two weeks. It will look more readable.
```
  .... plus(2, ChronoUnit.WEEKS)...
```

Reply:
will use 14 DAYS as WEEKS and above "will throw an UnsupportedTemporalTypeException." (javadoc)